### ISSUE TITLE
Psychonauts (Linux) cannot be installed or imported

### ISSUE THREAD

**Issue by krisutofu** [Heroic-Games-Launcher/HeroicGamesLauncher#3607]
### Describe the bug

It seems like this is either a regression of #2725 or just another instance of it that wasn't fixed. (Both have in common that they are doublefine production GOG games.)

The first time, I downloaded Psychonauts with Lutris.
When I switched to Heroic Games Launcher, importing Psychonauts did not work. At least I could install a fresh copy of it back then.

Now, when I accidentally deleted all of my personal computer configuration, it still cannot import the game files that Heroic Games Launcher previously had installed. It shows a success notification but immediately afterwards it switches the game's status back to uninstalled.

<details>
<summary>This is the Heroic general log portion which was generated by the failed import activity (click this line to view).</summary>

```
(04:14:35) INFO:    [ExtraGameInfo]:    Using cached ExtraGameInfo data for Psychonauts
(04:14:35) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/elmar/.config/heroic/gogdlConfig /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/elmar/.config/heroic/gog_store/auth.json auth
(04:14:35) INFO:    [ExtraGameInfo]:    Using cached ExtraGameInfo data for Psychonauts
(04:14:35) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
(04:14:35) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
(04:14:36) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
(04:14:36) DEBUG:   [Backend]:          {"free":167951790080,"diskSize":300941312000,"message":"156.42 GiB / 280.27 GiB","validPath":true,"validFlatpakPath":true}
(04:14:36) DEBUG:   [Backend]:          {"free":167951790080,"diskSize":300941312000,"message":"156.42 GiB / 280.27 GiB","validPath":true,"validFlatpakPath":true}
(04:14:43) INFO:    [Gog]:              Importing 1207658807: GOGDL_CONFIG_PATH=/home/elmar/.config/heroic/gogdlConfig /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/elmar/.config/heroic/gog_store/auth.json import /home/elmar/Games/gog/Psychonauts
(04:14:43) INFO:    [Backend]:          Preventing machine to sleep
(04:14:43) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
(04:14:43) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
(04:14:44) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/elmar/.config/heroic/gogdlConfig /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/elmar/.config/heroic/gog_store/auth.json auth
(04:14:44) ERROR:   [Gog]:              Game data falsy in getInstallInfo
(04:14:44) INFO:    [Backend]:          Adding shortcuts for Psychonauts
(04:14:44) INFO:    [Backend]:          imported Psychonauts
(04:14:44) INFO:    [Frontend]:         Refreshing gog Library
(04:14:44) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/elmar/.config/heroic/gogdlConfig /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/elmar/.config/heroic/gog_store/auth.json auth
(04:14:44) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
(04:14:44) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
(04:14:44) INFO:    [Gog]:              Getting GOG library
(04:14:44) INFO:    [Gog]:              Running command: GOGDL_CONFIG_PATH=/home/elmar/.config/heroic/gogdlConfig /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/elmar/.config/heroic/gog_store/auth.json auth
(04:14:45) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
(04:14:45) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
(04:14:45) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
(04:14:45) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
(04:14:46) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
(04:14:46) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
(04:14:46) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
(04:14:47) INFO:    [Gog]:              Saved games data
(04:14:47) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
(04:14:47) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
(04:14:47) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
(04:14:47) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
(04:14:47) INFO:    [Gog]:              Got install info from cache for 1207658807 on linux platform
```
</details>

It shows one error, the gamedata would be falsy.

Now, I can't import it and at 97% of downloading, it always stalls. The download freezes forever. I tried this 3 times, and every time it is the same. The logs just stop. Only the corresponding `*gogdl.log` file shows a "critical" error at the very end at ca. 97%:

```
[TASK_EXEC] CRITICAL: Task writer failed
[PROGRESS] INFO: = Progress: 96.71 5306243779/5486770546, Running for: 00:09:25, ETA: 00:00:19
[PROGRESS] INFO: = Downloaded: 4131.74 MiB, Written: 5060.43 MiB
[PROGRESS] INFO:  + Download	- 0.00 MiB/s (raw) / 0.00 MiB/s (decompressed)
[PROGRESS] INFO:  + Disk	- 0.00 MiB/s (write) / 0.00 MiB/s (read)
```

When this happens, all of the download metrics freeze. And it might happen that when I switch from download view to library view (after it froze), it shows Psychonauts being installed at 0% (it cannot read the progress somehow).

If there is a way how I could manually add the existing game directory (as a quick workaround), I would like to know it. (The native Linux game had various issues which I needed to fix manually so I wanted to use my old files anyways.)

[heroic_general_log.md](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/files/14468021/heroic_general_log.md)

### Add logs

```text
Please see the link above
```

### Steps to reproduce

1. start Heroic Games Launcher (and log into GOG account)
2. Click on Psychonauts game (not installed)
3. Click on "install"
4. Click on "import game"
5. Choose the directory that contains the game files

### Expected behavior

The progress bar finishes, the notification reads "Psychonauts – Game imported" and the game status is shown as installed.

It should not set the game's status back to uninstalled in order to allow for the game being configured and played.

### Heroic Version

Latest Stable

### System Information

- 8 × 11th Gen Intel® Core™ i5-1135G7 @ 2.40GHz
- Kernel Version: 6.7.6-zen1-1-zen (64-bit)

### Additional information

I mentioned initially, I accidentally deleted all my personal computer configuration. I could recover only very few files. It could be, I copied them back into the new heroic configuration directory (I think it only were log files, some of them were even corrupted). My settings and saved data in `~/.local/share/Psychonauts` are also gone.

When I started Heroic, it was like using it the first time and I didn't find any abnormalities (Psychonauts couldn't import before). Importing and running other games works as normal.

**Comment by imLinguin** [comment]
In regards to downloading I think I fixed this issue in the recent gogdl build. 

About importing - it's troublesome. In some cases we can't verify if the user provided a valid path that actually contains the expected game files. I'll try to find some solution to improve this situation though.

**Comment by krisutofu** [comment]
Ah, I see the [commit](https://github.com/Heroic-Games-Launcher/heroic-gogdl/commit/8a32d87bf53d4cd9570dd8220c6b5e61a4c9d7dc).

Okay, so does this mean, I only need to wait for my installed package (chaotic-aur) to be updated? [Yeah, it is still from February.](https://aur.archlinux.org/packages/heroic-games-launcher-bin). I have installed the latest version as available on (chaotic-)AUR.

Maybe I should add that the default Psychonauts installation from GOG does not provide a fully functional experience without modification to the directory contents. If the reason for a failed import is really due to mismatch with a GOG file list or something, I would wish this check could be turned off for specific games to allow importing nevertheless. (My GOG Linux native games come with a start.sh script in the root of the downloaded directory.)

I can't import Syberia games either (which I downloaded and play with Lutris). Redownloading would need me to transfer the saved game data (and hope, it would still work with Lutris). The Syberia games came downloaded with their own wine prefix (don't know if this is due to configuration).

Thank you for your great effort with Heroic Game Launcher.
